packages/kit/src/views/ReferFriends/pages/InvitedByFriend/components/InvitedByFriendContent.tsx

```
@@ -5,7 +5,11 @@ import { ETranslations } from '@onekeyhq/shared/src/locale';
 
 import { ReferralBenefitsList } from '../../../components';
 
-function InvitedByFriendContent() {
+function InvitedByFriendContent({
+  referralCode,
+}: {
+  referralCode?: string;
+}) {
```

ESLint

[error] Replace `⏎··referralCode,⏎}:·{⏎··referralCode?:·string;⏎` with `·referralCode·}:·{·referralCode?:·string·`

(prettier/prettier)